README.md

```diff
@@ -1,5 +1,10 @@
 # ![logo](https://quintilesims.github.io/layer0/static/logo_rs.png)
 
+[![Build Status](https://travis-ci.org/quintilesims/layer0.svg?branch=master)](https://travis-ci.org/quintilesims/layer0)  [![Version](https://img.shields.io/badge/version-0.9.0-brightgreen.svg)](https://github.com/quintilesims/layer0/releases/tag/v0.9.0)  [![Documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://quintilesims.github.io/layer0/)
```

Comment:
Further reflection has prompted me to postpone the creation of a `latest` tag. Pushing two tags for the same release would likely wreak havoc wherever our scripts call `git describe --tags`. There's a Stack Overflow question [here](http://stackoverflow.com/questions/8089002/git-describe-with-two-tags-on-the-same-commit) that seems to imply that `git describe --tags` will take the newest tag, which means that in theory we could be sure to create the version-specific tag **after** the `latest` tag, but that seems fragile and janky.

So for now, we'll keep our cool badges and require a little bit of manual attention during the release process (it's documented in RELEASE.md) until we figure something better out.

- Removing #183 from this PR
- Referencing #117 for the future